tools/autodeployment/deploy_stage.sh

```diff
@@ -3,7 +3,7 @@
 # Shutdown
 cd ../telescope
 
-docker-compose -f docker-compose-staging.yml down
+docker-compose -f docker-compose-production.yml down
```

Comment:
In @c3ho's PR, we can deal with this, but calling this file `deploy_stage.sh` is odd if we use `-production` here.

Reply:
I think we can try to merge this one first and then @c3ho can rebase and change the name of the file and make the necessary adjustments.

Reply:
I'm fine with that